src/transform/EditTransform.css

```diff
@@ -80,4 +80,11 @@ a.pagelib_edit_section_link {
 
 .pagelib-platform-android.pagelib-theme-dark.pagelib-compatibility-filter a.pagelib_edit_section_link:active:before {
   background-color: #111;
-}
\ No newline at end of file
+}
+
+.pagelib-platform-ios h3.section_heading span.pagelib_edit_section_link_container,
+.pagelib-platform-ios h4.section_heading span.pagelib_edit_section_link_container,
+.pagelib-platform-ios h5.section_heading span.pagelib_edit_section_link_container,
+.pagelib-platform-ios h6.section_heading span.pagelib_edit_section_link_container {
+    display: none !important;
```

Comment:
Sorry but it isn't clear why `important` priority is needed here and I didn't see any comments in the commit, the PR, or the ticket. Since we add these edit pencils ourselves, I'm hoping we can avoid specifying `important` priority. (This isn't just me hating on `important`, BTW, it's even in the [MediaWiki CSS recommendations](https://www.mediawiki.org/wiki/Manual:Coding_conventions/CSS#.21important)!)